src/FSharp.Core/SI.fs

```diff
@@ -55,63 +55,63 @@ type joule = newton metre
 
 /// The SI unit of power, radiant flux
 [<Measure>]
-type watt = joule / second       
+type watt = joule / second
 
 /// The SI unit of electric charge, amount of electricity
 [<Measure>]
-type coulomb = second ampere 
+type coulomb = second ampere
 
 /// The SI unit of electric potential difference, electromotive force
 [<Measure>]
-type volt = watt/ampere        
+type volt = watt / ampere
 
 /// The SI unit of capacitance
 [<Measure>]
-type farad = coulomb/volt
+type farad = coulomb / volt
 
 /// The SI unit of electric resistance
 [<Measure>]
-type ohm = volt/ampere       
+type ohm = volt / ampere
 
 /// The SI unit of electric conductance
 [<Measure>]
-type siemens = ampere/volt         
+type siemens = ampere / volt
 
 /// The SI unit of magnetic flux
 [<Measure>]
-type weber = volt second        
+type weber = volt second
 
 /// The SI unit of magnetic flux density
 [<Measure>]
-type tesla = weber/metre^2      
+type tesla = weber / metre^2
 
 /// The SI unit of inductance
 [<Measure>]
-type henry = weber/ampere        
+type henry = weber / ampere
 
 /// The SI unit of luminous flux
 [<Measure>]
-type lumen = candela        
+type lumen = candela
 
 /// The SI unit of illuminance
 [<Measure>]
-type lux = lumen/metre^2 
+type lux = lumen / metre^2
 
 /// The SI unit of activity referred to a radionuclide
 [<Measure>]
-type becquerel = second^-1       
+type becquerel = second^- 1
```

Comment:
Should the space be here after the `-`? I would expected it to either be consistent with `\` - `second ^ -1`, or `second^-1`, as a special case of power operator, but not `second^- 1`.
@dsyme @nojaf wdyt?

Reply:
Never seen this in F# code, so this most likely is a gap.

Reply:
Also a bug!

Reply:
I'm surprised the formatted code is accepted by the compiler :)